scipy/spatial/qhull.pyx

```diff
@@ -651,7 +651,7 @@ cdef class _Qhull:
 
                         if ncoplanar >= coplanar.shape[0]:
                             with gil:
-                                tmp = coplanar
+                                tmp = np.array(coplanar)
```

Comment:
This change adds a cast from memory view to ndarray  and a cast from ndarray to memoryview. which weren't there before. Could be contributing to the 10us slow down referred in #12379